client/signup/steps/woocommerce-install/step-store-address/index.tsx

```diff
@@ -0,0 +1,116 @@
+import { TextControl } from '@wordpress/components';
+import { useI18n } from '@wordpress/react-i18n';
+import { ReactElement } from 'react';
+import { useSelector } from 'react-redux';
+import { FormCountrySelect } from 'calypso/components/forms/form-country-select';
+import { LoadingEllipsis } from 'calypso/components/loading-ellipsis';
+import StepWrapper from 'calypso/signup/step-wrapper';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+import SupportCard from '../components/support-card';
+import { ActionSection, StyledNextButton } from '../confirm';
+import {
+	useSiteSettings,
+	WOOCOMMERCE_STORE_ADDRESS_1,
+	WOOCOMMERCE_STORE_ADDRESS_2,
+	WOOCOMMERCE_STORE_CITY,
+	WOOCOMMERCE_DEFAULT_COUNTRY,
+	WOOCOMMERCE_STORE_POSTCODE,
+} from '../hooks/use-site-settings';
+import useWooCommerceOnPlansEligibility from '../hooks/use-woop-handling';
+import type { WooCommerceInstallProps } from '..';
+import type { ChangeEventHandler } from 'react';
+import './style.scss';
+
+export default function StepStoreAddress( props: WooCommerceInstallProps ): ReactElement | null {
+	const { goToStep, isReskinned, headerTitle, headerDescription } = props;
+	const { __ } = useI18n();
+
+	const siteId = useSelector( getSelectedSiteId ) as number;
+
+	const { wpcomDomain } = useWooCommerceOnPlansEligibility( siteId );
+
+	const { get, save, update, countriesList } = useSiteSettings( siteId );
+
+	const handleCountryChange: ChangeEventHandler< HTMLInputElement > = ( event ) => {
+		update( WOOCOMMERCE_DEFAULT_COUNTRY, event.target.value );
+	};
+
+	function getContent() {
+		return (
+			<>
+				<div className="step-store-address__info-section" />
+				<div className="step-store-address__instructions-container">
+					<TextControl
+						label={ __( 'Address line 1', 'woocommerce-admin' ) }
+						value={ get( WOOCOMMERCE_STORE_ADDRESS_1 ) }
+						onChange={ ( value ) => update( WOOCOMMERCE_STORE_ADDRESS_1, value ) }
+					/>
+
+					<TextControl
+						label={ __( 'Address line 2', 'woocommerce-admin' ) }
+						value={ get( WOOCOMMERCE_STORE_ADDRESS_2 ) }
+						onChange={ ( value ) => update( WOOCOMMERCE_STORE_ADDRESS_2, value ) }
+					/>
+
+					<FormCountrySelect
+						name="country_code"
+						value={ get( WOOCOMMERCE_DEFAULT_COUNTRY ) }
+						onChange={ handleCountryChange }
+						countriesList={ countriesList }
+						translate={ __ }
```

Comment:
Does `FormCountrySelect` not get `translate` from [`localize`](https://github.com/Automattic/wp-calypso/blob/171ef42853484467b2bc4cf575547d9e41f8057c/client/components/forms/form-country-select/index.jsx#L67)?

Reply:
[nop, it doesn't](https://github.com/Automattic/wp-calypso/blob/40853249648188a38f4ccbdb205f8e2b108a7c7f/client/components/forms/form-country-select/index.jsx#L16). The `translate` is a component property :-/

Reply:
let me check... 😓

Reply:
It's weird. When I remove the prop I get the error:

![image](https://user-images.githubusercontent.com/77539/148443446-50afadbc-62e7-4132-a388-aa4f426a5c37.png)

Reply:
Addressed [here](https://github.com/Automattic/wp-calypso/pull/59505/commits/71c78d69e64f442cab288f23e7c63296bea28bf0). Sorry, and thanks!